pkg/webhook/controlplane/ensurer.go

```diff
@@ -130,14 +153,23 @@ var (
 		MountPath: "/etc/kubernetes/audit-override",
 		ReadOnly:  true,
 	}
-	auditPolicyVolume = corev1.Volume{
+	// default audit policy; can be overridden with custom audit policy
+	defaultAuditPolicyVolume = corev1.Volume{
 		Name: metal.AuditPolicyName,
 		VolumeSource: corev1.VolumeSource{
 			ConfigMap: &corev1.ConfigMapVolumeSource{
 				LocalObjectReference: corev1.LocalObjectReference{Name: metal.AuditPolicyName},
 			},
 		},
 	}
+	customAuditPolicyVolume = corev1.Volume{
+		Name: metal.AuditPolicyName,
+		VolumeSource: corev1.VolumeSource{
+			ConfigMap: &corev1.ConfigMapVolumeSource{
+				LocalObjectReference: corev1.LocalObjectReference{Name: "custom-audit-policy"},
```

Comment:
maybe also a constant like metal.CustomAuditPolicyName

Reply:
Will do that in the appropriate branch!